### ISSUE TITLE
CSS: ordre des propriétés et valeurs préfixées

### ISSUE THREAD

**Issue by YannisDelmas** [YannisDelmas/beau-code-web#26]
Utiliser une propriété expérimentale présente toujours un risque. Toutefois, si on est prêt à l'assumer…

Il faudrait mettre en premier les propriétés/valeurs les plus anciennes (préfixées, donc) et en dernier celles non préfixées.

Cf. <https://stackoverflow.com/questions/36225219/what-is-the-proper-way-to-order-vendor-prefixes>

Bien noter, quand même, que la plupart des préfixes n'ont plus réellement d'intérêt aujourd'hui. Seulement de compatibilité avec de (très) vieux navigateurs.

**Comment by enguerranws** [comment]
Est-ce qu'on peut mentionner à ce sujet, sans les recommander, des outils comme autoprefixer pour faire ce travail ?

**Comment by YannisDelmas** [comment]
Oui, [autoprefixer](https://github.com/postcss/autoprefixer), ça peut être une bonne solution pour certains projets pour qui postCSS est admissible. Il y a des équivalents pour LESS etc.

Quand il n'y a pas grand-chose à préfixer, ce qui devrait être le cas pour la plupart des projets, le mieux est probablement de faire ça directement dans le CSS, même si ça diminue un peu la lisibilité. Dans ce cas, la [version en ligne d'autoprefixer](https://autoprefixer.github.io/) est peut-être la plus indiquée, non?

**Comment by enguerranws** [comment]
Oui, je pensais surtout à la version en ligne de l'outil, sans parler
forcément de ses implémentations dans les pré/post processeurs.

Le jeu. 5 mai 2022 à 11:33, Yannis Delmas ***@***.***> a
écrit :
